README.md

```diff
@@ -5,6 +5,8 @@
   <img src='https://github.com/americanexpress/one-app/raw/master/one-app.png' alt="One App - One Amex" width='50%'/>
 </h1>
 
+[![Slack](https://img.shields.io/badge/slack-join%20chat-blue)](https://join.slack.com/t/one-amex/shared_invite/enQtOTA0MzEzODExODEwLTlmYzI1Y2U2ZDEwNWJjOTAxYTlmZTYzMjUyNzQyZTdmMWIwZGJmZDM2MDZmYzVjMDk5OWU4OGIwNjJjZWRhMjYV)
```

Comment:
when i follow that url is comes up as "This invite link is no longer active."

Reply:
weird... It worked for me yesterday

Reply:
Thanks for checking @JAdshead  yeah the link wasn't working! i've updated it